bedrock/firefox/tests/test_base.py

```diff
@@ -419,63 +419,3 @@ def test_fx_firstrun_dev_edition_legacy_redirect(self, render_mock):
         assert resp['location'].endswith('/firefox/new/')
 
 
-@patch('bedrock.firefox.views.l10n_utils.render', return_value=HttpResponse())
-class TestTrackingProtectionTour(TestCase):
-    def setUp(self):
-        self.view = fx_views.TrackingProtectionTourView.as_view()
```

Comment:
Please clean up the views these tests are associated with:

https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/views.py#L601-L626